### PR TITLE
Body bags and the things to fill them

### PR DESCRIPTION
**Pull request by skraaak**
## About The Pull Request
A quick and dirty bit of code to wildly unrelated items.
<hr>

Fixes the behavior of bodybags and stasisbags by adding "closed" and "full" states as separate sprites.

Fixed some minor sprite issues with several handguns and related items and replaced an ancient bay gun with a modern Eris one.

Tested _extensively_ on local build.
<hr>
</details>

## Changelog
:cl:
Fixed: Stasis and body bags, they now have a "full" icon state that applies when a body, living or dead is inside them.
Edited: Stasis bags now are animated to indicate their statuses, "Idle, working, used" to make them more clear to use and to make them seem more urgent and technological. 
Minor changes to the M1911 sprites to clean them up and polish them to final grade.
Flipped the .44 pistol magazines to face the correct direction. 
Cleaned up the new M6 style Lamia sprites and gave the "Advanced" version an empty indicator light, and cleaned up the code for it.
Changed descriptions and names of the M1911s to match FS standards.
Replaced the Desert Eagle with a modified version of the new Eris Master "bolter style" Lamia with working ammo counter and new magazines. 
Increased magazine capacity of the "Deagle" to 8 instead of 7, largely for sprite consistency. 
Minor description fixes for absolutely ancient issues.
Added: Added FS HG .45 "Bronco" handguns and ammunition to IH MedSpec lockers, giving them a less extreme and more "classy" alternative to machinepistols.
Placed a roll of police tape in the IHC's locker.

/:cl:

### CODE REVIEW

File: code/game/objects/items/bodybag.dm [hunk 1]
```diff
@@ -34,6 +34,9 @@
 	desc = "A plastic bag designed for the storage and transportation of cadavers."
 	icon = 'icons/obj/bodybag.dmi'
 	icon_state = "bodybag"
+	icon_state = "bodybag_closed"
+	icon_state = "bodybag_full"
+	icon_state = "bodybag_open"
```
Comment (PolymorphBlue) [inline code/game/objects/items/bodybag.dm]: No point setting icon_state multiple times like this, it'll just use the last one.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -150,6 +151,10 @@
 	new /obj/item/weapon/gun/projectile/automatic/IH_machinepistol(src)
 	new /obj/item/weapon/storage/pouch/pistol_holster(src)
 	new /obj/item/weapon/storage/briefcase/crimekit(src)
+	new /obj/item/ammo_magazine/c45m/rubber(src)
+	new /obj/item/ammo_magazine/c45m/rubber(src)
+	new /obj/item/ammo_magazine/ammobox/c45/rubber(src)
+	new	/obj/item/weapon/gun/projectile/colt/IH_colt(src)
```
Comment (PolymorphBlue) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Spacing

File: code/game/objects/items/bodybag.dm (hunk 1 above)
Comment (skraaak) [reply code/game/objects/items/bodybag.dm]: > No point setting icon_state multiple times like this, it'll just use the last one.
>Spacing
Both fixed. The spacing is an anomaly that crops up often in the closet dms where DM shows it as a space, but other editors see a Tab. Doesn't affect the code, just looks unsightly, but I fixed all instances I found anyway.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 2 above)
Comment (skraaak) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Fixed.

**Comment by Kazkin** [comment]
Good to go fam-a-lam. Also, love the deagle changes.

**Comment by APWill** [comment]
24 hours have passed.

za warudo

**Pull request merged**